include/benchmark/benchmark.h

```diff
@@ -804,6 +805,12 @@ class Benchmark {
   // REQUIRES: The function passed to the constructor must accept an arg1.
   Benchmark* DenseRange(int64_t start, int64_t limit, int step = 1);
 
+#if defined(BENCHMARK_HAS_CXX11)
+  // Run this benchmark once for every value in list.
+  // REQUIRES: The function passed to the constructor must accept an arg1.
+  Benchmark* Explicit(std::initializer_list<int64_t> list);
```

Comment:
Uhm, are we not crippling ourselves by using `std::initializer_list` here?
I agree that it should be ArrayRef/Span, but elsewhere here we seem to take `std::vector`.

Reply:
yes. given the `initializer_list` requires brace initialization and a `vector` has an implicit constructor that takes an `initializer_list`, we should err on taking a `vector` here.